squid/templates/trust-manager-bundle.yaml

```
@@ -10,9 +10,14 @@ spec:
   - secret:
       name: "{{ .Values.namespace.name }}-root-ca-secret"
       key: "ca.crt"
-  - useDefaultCAs: true
   target:
     configMap:
       key: "ca-bundle.crt"
-    namespaceSelector: {}      
-{{- end }} 
+    {{- if .Values.test.enabled }}
+    namespaceSelector: {}
+    {{- else }}
+    namespaceSelector:
+      matchLabels:
+        konflux-ci.dev/type: tenant
+    {{- end }}
```

```suggestion
    namespaceSelector: {{ .Values.trust-manager.namespaceSelector }}
```

Need to verify that the default value could be:
```
      matchLabels:
        konflux-ci.dev/type: tenant
```
We have yaml objects as values in helm charts. It should work in one way or another.